<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQBoardsSettings</class>
 <widget class="QWidget" name="QQBoardsSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListView" name="bouchotListView"/>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="addBouchotButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../rc/quteqoin_img.qrc">
          <normaloff>:/img/add-icon.png</normaloff>:/img/add-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteBouchotButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../rc/quteqoin_img.qrc">
          <normaloff>:/img/delete-icon.png</normaloff>:/img/delete-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editBouchotButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../rc/quteqoin_img.qrc">
          <normaloff>:/img/menu-edit-icon.png</normaloff>:/img/menu-edit-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../rc/quteqoin_img.qrc"/>
 </resources>
 <connections/>
</ui>
